Update mdcb-components.md (#2487)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-multi-data-centre/mdcb-components.md

## Control Plane
{{< img src="/img/mdcb/mdcb-control-plane.png" width="800" height="975" alt="The Tyk Control Plane" >}}
(TODO: rename to "Tyk Gateway")

The Control Plane must consist of the following elements:
- **Tyk Dashboard** (used to configure and control the whole Tyk installation)
- **Tyk Gateway** (used for creation of keys and certificates, this does not service API requests; it is important to ensure there is no public access to it and it must not be sharded (tagged) as it "belongs" to the whole Tyk installation)

> - **Tyk MDCB** (used for propagation configurations to Data Plane gateways)

Reviewer: ```suggestion
- **Tyk MDCB** (used for propagation of configurations - such as APIs and policies - to Data Plane gateways)
```

> - **Redis** (in-memery data store and message broker. High availability Redis data store that should be backed up in case of failure; this [document](https://redis.io/docs/management/persistence/) gives recommendation on Redis persistency)

Reviewer: ```suggestion
- **Redis** (in-memory data store and message broker; a high availability Redis data store that should be backed up in case of failure; this [document](https://redis.io/docs/management/persistence/) gives recommendation on Redis persistency)
```

> - **MongoDB or SQL** (a persistent data store for configurations and analytics. It that should be deployed and set up for redundancy and high availability)

Reviewer: ```suggestion
- **MongoDB or SQL** (a persistent data store for configurations and analytics; it should be deployed and set up for redundancy and high availability)
```
